locust-tools/locustfile.py

```diff
@@ -0,0 +1,127 @@
+from locust import HttpLocust, TaskSet, task
+import requests
+import os
+import random
+
+def find_metrics(host, app_name):
+    r = requests.get('{}/api/v1/series?match[]={{job="{}"}}'.format(host, app_name)).json()
+    return r['data']
+
+metrics = find_metrics(<<prometheus url>>,<<prometheus job>>)
```

Comment:
Yep it was a time thing since we not sure how we was going to be running it initially, this however is not priority at the moment and it is something that we could possibly test.